app/helpers/fraud_indicators_helper.rb

```diff
@@ -11,11 +11,21 @@ def link_to_indicator_list(fraud_indicator)
              return nil
            end
     table = fraud_indicator.list_model_name.split("::").last.downcase
+    path_with_type = "hub_#{type}_#{table}s_path"
+    path_without_type = "hub_#{table}s_path"
 
-    if respond_to?("hub_#{type}_#{table}s_path")
-      link_to send("hub_#{type}_#{table}s_path") do
-        content_tag :i, "list_alt", class: "icon-"
-      end
-    end
+    matching_path = nil
+    matching_path = path_with_type if respond_to?(path_with_type)
```

Comment:
if the indicator type is not in safelist, you have to maintain a list of safe things that it can check against to see if the value is not in that list.